add utilization rate note & version compatibility (#983)
vesoft-inc/nebula-docs: docs-2.0/nebula-dashboard/6.monitor-parameter.md

This topic will describe the monitoring metrics in Nebula Dashboard.

## Machine

!!! note

>     - The operating system of the monitored machine is Linux by default.

Reviewer: Same as preceding.

>     - The operating system of the monitored machine is Linux by default.

Author: ```suggestion
    - All the machine metrics listed below are for the Linux operating system.
```
